software_development/BUILDING.md

```diff
@@ -97,3 +97,26 @@ For changes related to the user interface in the `web-ui` module or the metadata
 ```
 mvn process-resources
 ```
+
+Tool chain
+----------
+
+GeoNetwork requires Java 8 at this time. If you have multiple JDK environments installed
+our build can make use of an optional `~/.m2/toolchain.xml` file.
```

Comment:
From the `pom.xml` it should be `~/.m2/toolchains.xml` instead of `~/.m2/toolchain.xml`

I tested the configuration and seem pick up, but the startup error about the version occurs if the default version is 11:

```
[INFO] --- maven-toolchains-plugin:3.0.0:toolchain (default) @ web-app ---
[INFO] Required toolchain: jdk [ version='8' ]
[INFO] Found matching toolchain for type jdk: JDK[/PATH/candidates/java/8.0.201-oracle]

...

[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ web-app ---
[INFO] Toolchain in maven-compiler-plugin: JDK[/PATH/candidates/java/8.0.201-oracle]
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 11 source files to /core-geonetwork/web/target/test-classes
...
2020-10-23 07:58:42,184 FATAL [jeeves.engine] -    Exception : java.lang.RuntimeException: Java 8 required, cannot start with "11.0.8"
2020-10-23 07:58:42,189 FATAL [jeeves.engine] -    Message   : Java 8 required, cannot start with "11.0.8"
2020-10-23 07:58:42,192 FATAL [jeeves.engine] -    Stack     : java.lang.RuntimeException: Java 8 required, cannot start with "11.0.8"
	at jeeves.config.springutil.JeevesContextLoaderListener.contextInitialized(JeevesContextLoaderListener.java:63)

```

Apart of the toolchain stuff, the change works fine, starts up fine with Java 8 and throws the exception if other version is used.

Reply:
That is odd, the surefire plugin must be picking up the wrong JDK. Let me check version numbers on the surefire plugin (to ensure it is new enough to be aware of tool chain).

Reply:
This is working for me locally, tried updating the surefire plugin to 3.0.0-M5 and that still works.

Still, since this is not important, I have pushed it up as two commits to the PR. You can accept (since toolchains.xml is optional) or just cherry-pick the one change.